tests/model_registry/utils.py

```diff
@@ -0,0 +1,62 @@
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.namespace import Namespace
+from ocp_resources.service import Service
+from ocp_resources.model_registry import ModelRegistry
+from kubernetes.dynamic.exceptions import ResourceNotFoundError
+
+from tests.utils import ProtocolNotSupported
+from utilities.constants import Protocols
+from tests.model_registry.constants import ModelRegistryEndpoints
+
+
+def get_mr_service_by_label(client: DynamicClient, ns: Namespace, mr_instance: ModelRegistry) -> Service:
```

Comment:
Can we use `ensure_exists` here?

Reply:
I'm not sure what you're suggesting. This function would raise if the Service didn't exist?

Reply:
Yes, it would raise if the service doesn't exist. I'm not sure if you could use it here or not --feel free to resolve this comment if you can't use it.